src/systemctl/systemctl.c

```diff
@@ -4523,6 +4523,10 @@ static int print_property(const char *name, sd_bus_message *m, const char *conte
         return 0;
 }
 
+struct property_info {
+        const char *load_state, *active_state;
+} property_info = {};
```

Comment:
sorry I dont's know how to cross the comments. I am editing it . Let's keep the definition global and variables for this local.